Z/z3/build_tarballs.jl

```diff
@@ -69,7 +70,7 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     BuildDependency("libjulia_jll"),
-    Dependency(get_addable_spec("GMP_jll", v"6.2.0+5")),
+    Dependency("GMP_jll"),
```

Comment:
We still have to compile with an old GMP, because of compatibility reasons.

Reply:
Which GMP should I use? The previous `6.2.0+5` did not work.

Reply:
I'd need to understand _why_ it didn't work.

Reply:
The log message was
```
┌ Warning: Dependency GMP_jll does not have an (Stdlib)Artifacts.toml in /Users/eschnett/.julia/packages/GMP_jll/IA6Pi!
└ @ BinaryBuilderBase ~/.julia/dev/BinaryBuilderBase/src/Prefix.jl:742
```

Afterwards, no GMP was installed, and cmake failed.

Reply:
I used Julia 1.8.5 and the `master` versions of `BinaryBuilder` and `BinaryBuilderBase`.